embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParser.java

```diff
@@ -0,0 +1,112 @@
+package org.embulk.spi.time;
+
+import org.jruby.runtime.ThreadContext;
+
+import java.util.HashMap;
+import java.util.List;
+
+import static org.embulk.spi.time.StrptimeParser.FormatBag.has;
+import static org.jruby.RubyRational.newRationalCanonicalize;
+
+/**
+ * This class has {@code StrptimeParser} and provides methods that are calls from JRuby.
+ */
+public class RubyDateParser
+{
+    private final ThreadContext context;
```

Comment:
Thank you for your advice. Fixed it on b9903b3690c08faace8ddfe1420a8478de4d038e. In the fix, `context` field is removed in the class and each method receives the `context` as 1st argument. If the fix is not expected, please give me an example code that you expected.